cmd/sidecar/main.go

```diff
@@ -30,33 +30,34 @@ func main() {
 	tokenFilePath := flag.String("t", "/run/conjur/access-token",
 		"Path to Conjur access token")
 
-	configAuthn, err := authnConfig.NewFromEnv(clientCertPath, tokenFilePath)
+	// Create new Storage
+	configStorage, err := storageConfig.NewFromEnv(tokenFilePath)
```

Comment:
don't we want to read this as an env var instead? this can go inside `storage/config`